packages/sdk/identity/package.json

```diff
@@ -18,7 +18,7 @@
     "build": "tsc -b .",
     "clean": "tsc -b . --clean",
     "docs": "typedoc",
-    "test:reset": "yarn --cwd ../../protocol devchain generate-tar .tmp/devchain.tar.gz --migration_override ../../dev-utils/src/migration-override.json --upto 25",
+    "test:reset": "yarn --cwd ../../protocol devchain generate-tar .tmp/devchain.tar.gz --migration_override ../../dev-utils/src/migration-override.json --upto 26",
```

Comment:
wow kinda crazy this complicated command is in so many packages. I wonder if we could have it just in protocol and then call that from each?

Reply:
@m-chrzan what do you think?

Reply:
I don't think you have to fix this in this PR, but if you want you could open a new one fixing this